tests/regression_tests/deplete_with_transport/test.py

```diff
@@ -134,6 +138,18 @@ def test_full(run_in_tmpdir, problem, multiproc):
     # Check that no additional tallies are loaded from the files
     assert np.all(n_tallies == 0)
 
+    # Check that runtimes are qualitatively correct
+    assert runtimes['reading cross sections'][0] != 0
```

Comment:
Looks like the tests below rely on the values in runtimes being arrays:
```suggestion
    # Convert values in runtimes to arrays
    runtimes = {k: np.array(v) for k, v in runtimes.items()}

    # Check that runtimes are qualitatively correct
    assert runtimes['reading cross sections'][0] != 0
```